index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Nimplay</title>
+    <link rel="shortcut icon" href="./Images/nimplayLogo.jpg">
+    <link rel="stylesheet" href="style.css">
+    
+</head>
+<body>
+    <header>           
+        <div class="bar2">
+            <div class="logo">
+              <p id="name">Nimrod Acosta</p> 
+            </div>
+            <div class="burger">
+                <div class="normalb">
+                  <img src="/Images/Union.png" class="union" alt="">                    
+                </div>
+            </div>
+        </div>
+        
+
+    </header>
+    <section class="continer">
+        <h1 class="title">Hey There. I’m Nimrod</h1>
+        <h2 class="subtitle">I’am a Software Developer</h2>
+        <p class="paragraph">Encoding for life. <br> 
+          Web Solutions and Mobile Applications</p>
+        <div class="social">
+            <div class="socialimg">
+              <img src="Images/Github.png" alt="">
+            </div>
+            <div class="socialimg">
+              <img src="Images/Icon-Linkedin.png" alt="">
+            </div>
+            <div class="socialimg">          
+              <img src="Images/hand.png" alt="">
+            </div>
+            <div class="socialimg">
+              <img src="Images/Icon-Twitter.png" alt="">
+            </div>
+            <div class="socialimg">
+              <img src="Images/Icon-Medium.png" alt="">
+            </div>
+        </div>
+        <img src="/Images/Illustration.png" alt="" id="ilustration">       
```

Comment:
- [ ] Please remove this background image from here (HTML file) and add it as `background-image` property in your style sheet `style.css` for semantic reasons. You can use this [link ](https://www.w3schools.com/cssref/pr_background-image.asp)to know more about it and this [link](https://www.w3schools.com/cssref/pr_background-position.asp) to know how to position it.